api/libproject/devstart.rb

```diff
@@ -436,8 +436,7 @@ def add_options
   end
 
   def validate_options
-    super
-    if @opts.version == nil
+    if @opts.project == nil || @opts.account == nil ||@opts.version == nil
```

Comment:
Should we just change the base validation to allow both? Maybe print a warning that says "ignoring account since you also passed creds_file"?

Reply:
I think I'd prefer to leave it this way... for all the commands but this one, passing both makes no sense, and it's a bit clearer to indicate that loudly if someone tries.

(But I don't have a super-strong feeling either way.)